src/foam/util/Emails/EmailsUtility.java

```diff
@@ -86,7 +88,7 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
     // SERVICE CALL: to fill in email properties.
     EmailPropertyService cts = (EmailPropertyService) x.get("emailPropertyService");
     try {
-      cts.apply(x, group, emailMessage, templateArgs);
+      cts.apply(x, group, emailMessage, templateArgs, theme);
```

Comment:
I think if we already have the supportConfig, we can apply it here then SupportConfigEmailPropertyService is no longer needed.

Reply:
yeah, I think we better setup all email things in EmailUtility